wire-ios-data-model/Source/MLS/MLSActionExecutor.swift

```diff
@@ -51,6 +52,7 @@ public actor MLSActionExecutor: MLSActionExecutorProtocol {
     private let coreCryptoProvider: CoreCryptoProviderProtocol
     private let commitSender: CommitSending
     private var continuationsByGroupID: [MLSGroupID: [CheckedContinuation<Void, Never>]] = [:]
+    private let onNewCRLsDistributionPointsSubject = PassthroughSubject<CRLsDistributionPoints, Never>()
```

Comment:
I'm really no Combine expert so maybe this is bad practice but couldn't we inject the publisher into all the places where we need to report new CRLs? Then we would only need to subscribe once?

Reply:
created a ticket to improve this later https://wearezeta.atlassian.net/browse/WPB-6673